pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCaseDatawithEstimations.py

```diff
@@ -33,6 +33,7 @@
 
 class TestGetCaseDatawithEstimations(fake_filesystem_unittest.TestCase):
     path = '/home/CaseEstimationData'
+    gd.Conf.v_level = 'Debug'
```

Comment:
The test 'test_except_non_existing_files' checks prints on debug level. I didn't want to change the test itself, so I changed the verbosity level for this class. I now changed it, so that exclusively the test where it is needed uses the debug level.